lib/spack/docs/index.rst

```diff
@@ -37,21 +37,24 @@ package:
 If you're new to spack and want to start using it, see :doc:`getting_started`,
 or refer to the full manual below.
 
------------------
-Table of Contents
------------------
-
 .. toctree::
    :maxdepth: 2
+   :caption: Documentation and Tutorial
 
    features
    getting_started
    basic_usage
+   workflows
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Reference Manual
+
    packaging_guide
    mirrors
    configuration
    developer_guide
-   case_studies
    command_index
    package_list
    API Docs <spack>
```

Comment:
In my mind, the distinction is pretty clear. Spack "Users" are people who only run Spack commands. Spack "Developers" are people who edit packages, core Spack libraries, or documentation, and then contribute that code back to Spack. I don't think the distinction between "Tutorial" and "Reference Manual" is any more clear.

What about something like this?:
```rst
.. toctree::
   :maxdepth: 2
   :caption: Tutorials

   features
   getting_started
   basic_usage
   workflows

.. toctree::
   :maxdepth: 2
   :caption: Reference Manual

   configuration
   mirrors
   package_list
   command_index

.. toctree::
   :maxdepth: 2
   :caption: Contributing to Spack

   contribution_guide
   packaging_guide
   developer_guide
   API Docs <spack>
```